webpack/assets/javascripts/react_app/stores/HostsStore.js

```diff
@@ -1,62 +0,0 @@
-import AppDispatcher from '../dispatcher';
-import {ACTIONS} from '../constants';
-import AppEventEmitter from './AppEventEmitter';
-
-// Internal object of Hosts
-const _hosts = {};
-
-class HostsEventEmitter extends AppEventEmitter {
-  constructor() {
-    super();
-  }
-
-  getHostData(id) {
-    _hosts[id] = _hosts[id] || { data: [] };
-
-    return _hosts[id];
-  }
-
-  // workaround for max listeners
-  maxListers(selector = 'meta[name=itemsPerPage]') {
-    const element = document.querySelector(selector);
-
-    if (element && element.content) {
-      // * 2 as we have both success and failure listeners
-      return (element.content * 2);
-    }
-
-    return 40;
```

Comment:
could you also remove the meta attribute that was introduced as part of the original PR - see https://github.com/theforeman/foreman/commit/901fe896ae50ed3c75ee0a86bc78463f9973dc76#diff-7eb571b855131f643240b7b93ddfd44dR3